README.md

```diff
@@ -16,14 +16,22 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
+        * Any applicable egress charges are generally outside the purview of Fornax (more in the "aside" below). The exception would be egress on data that the user exports from the platform (download to local machine, send to a bucket, etc.). I don't know the Fornax configuration to know specifics, or whether it's something that developers and users need to be aware of.
+            * Aside:  Historically, the data holder (e.g., IRSA) typically covers the cost (e.g., egress) of delivering the data to the user, but cloud storage buckets are starting to change both the workflows and the cost models. Buckets support large-scale data access in ways that an archive like IRSA cannot support from on-premise. This is great, but also means more data requests and larger (and less predictable) costs. Data holders can often get the costs covered through grants, arrangements with the cloud provider (e.g., AWS), etc. But, in some cases they will decide that the best option is to make the data available in a bucket under what I would call the "cloud-pricing model" and sum up as "everyone pays for what *they* use". In particular, this means the data holder will pay the *storage* costs (which they can predict and easily control), and the requestor/end user will pay the *access* costs including egress (which the data holder cannot easily predict or control, but the requestor can). Individual charges are generally small and reasonable when spread around in this way, and the the cloud provider often offers free access up to some small but reasonable limit. However, egress is a particular fee that often does not apply at all, but in other cases can balloon to $$$ very quickly. So it is absolutely something to be considered, planned for, and controlled. Lastly, who pays the access/egress costs is determined by a setting on the bucket. If the bucket is "requestor pays", the user/requestor will need (e.g.,) AWS credentials to access it -- charges are then billed to the AWS account that owns the credentials.
+        * As far as I can tell, the best (AWS) option for actual controls -- beyond just "monitoring" support -- is [AWS Budgets Actions](https://aws.amazon.com/blogs/aws-cloud-financial-management/get-started-with-aws-budgets-actions/).
+        * AWS may charge *ingress* fees to bring data into an SMCE pod or user instance. This would be completely separate from any egress fees. There Someone working more directly on Fornax Daskhub would need to answer whether/how ingress applies.
 
 ## Getting started
 * How do I get an account?
 * How to Log in?
   * Log in at  https://daskhub.fornaxdev.mysmce.com/
-* How to log out?
+* How to end a session?
   *  go to `File` Menu: `hub control panel` and `stop my server`.  This insures the server is not still running when you logout which would be a waste of resources
   *  then logout in the upper right
```

Comment:
Suggesting "end a session" just to try to emphasize that I think the important thing here is that they do the "stop my server" step. Whether they actually log out is not important, as far as I know.

(This could be made more obvious and intuitive on the platform, perhaps by providing "stop my server" in the File menu directly. I don't think users will naturally realize they (1) *should* stop the server, and (2) "hub control panel" is the right, first button to push to do that. But that's a different issue and repo.)